### DISCUSSION TITLE
Resources for getting started

### DISCUSSION THREAD

**Discussion by peytonrunyan** [cozodb/cozo#117] [General]
Hi there! I'm working through the tutorial at the moment but kind of struggling just to establish a conceptual framework to even think about these things. I've worked with RDF triples before briefly, and have plenty of experience with relational databases. Is there a resource that I can read through or a video for how to best conceptualize the organization and relationship of data in a way that make the best use of Cozo?

**Comment by zicklag** [comment]
I've just discovered Cozo 2 days ago, and I spent a day getting more familiar with it. I had a similar issue with the tutorial seeming a little hard to map to real-life database use-cases.

I think it's partly due to the use of a lot of small examples with variable names like `a`, `b`, `c`, in a way that makes it hard to understand the practical value in a normal scenario.

There's only a slim chance that I'll get to it, but I was considering creating a more approachable tutorial for the docs. Tutorials is something I think I've had good success writing before, and I like to make things understandable, so I might try to squeze out some time for it, but no promises. :grin:

**Reply by peytonrunyan** [reply]
That would be wonderful! The other thing that I just realized is that the docs that explain all of the concepts come after the tutorial. I didn't realize from the sidebar that those weren't sub-sections of the tutorial. I'm going to give those a look and see if it doesn't clear things up.

<img width="650" alt="Screenshot 2023-05-16 at 8 31 39 PM" src="https://github.com/cozodb/cozo/assets/44583861/68b9bb35-36d2-446c-b08b-ec59da74fb06">
